ui/src/app/components/resource-card.tsx

```diff
@@ -162,14 +162,18 @@ export class ResourceCard extends React.Component<Props, State> {
   }
 
   get actionsDisabled(): boolean {
-    return !this.writePermission;
+    return !this.writePermission && !this.isConceptSet;
```

Comment:
Is it worth calling this out specifically for concept sets? I think its okay for us to enable the dropdown and have nothing inside it enabled for other resources as well.

Reply:
I called it out for only concept sets because I was concerned that it would be confusing to have dropdowns that open but don't let you do anything. If you think it would be ok, I'll change it, then we don't have a magic conditional gating this.